### PR TITLE
[DFFE-3]Develop a Contact form page using HTML

### PR DESCRIPTION
**Pull request by Nutan2305**

### CODE REVIEW

File: submissions/Nutan/index.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  <label for="fname">Full Name:</label><br>
+  <input type="text" id="fname" name="fname" placeholder="Enter your full name" size="22" ><br><br>
+  <label for="email">Email:</label><br>
+  <input type="text" id="email" name="email" placeholder="Enter your email address" size="22"><br><br>
+  <label for="subject" >Subject:</label><br>
+  <input type="text" id="sub"  name="sub" placeholder="Message subject" size="22"><br><br>
+  <label for="message" >Message</label><br>
+  <input type="text" id="message" name="message" style="height:120px; width:500px; "><br><br>
+  <input type="submit" value="Submit">
+</form>
```
Comment (TundeOladejo) [inline submissions/Nutan/index.html:21]: Format this block properly

File: submissions/Nutan/index.html [hunk 2]
```diff
@@ -0,0 +1,23 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>document</title>
+</head> 
+<body style="margin-left: 20px;margin-top:20px;margin-right: 20px; background-color: paleturquoise">
```
Comment (TundeOladejo) [inline submissions/Nutan/index.html:8]: You are not expected to use any CSS styling for this task. Kindly go through and make changes in other places too.

**Review by TundeOladejo — CHANGES_REQUESTED** [review]

**Review by adaugochi — DISMISSED** [review]
This task should be inside `html-forms` folder. This `html-forms` folder should be inside your name folder

File: submissions/Nutan/html-forms/index.html [hunk 3]
```diff
@@ -0,0 +1,26 @@
+<html lang="en">
```
Comment (adaugochi) [inline submissions/Nutan/html-forms/index.html:1]: this file name `index.html` doesn't tell what this file is for. Kindly name it to something like `contact.html`

**Review by Ireti203 — APPROVED** [review]

**Pull request merged**